mpas_analysis/ocean/nino34_index.py

```diff
@@ -154,7 +153,11 @@ def compute_nino34_index(regionSST, calendar):  # {{{
 
     anomaly = regionSST.groupby('month') - monthlyClimatology
 
-    return _running_mean(anomaly.to_pandas())  # }}}
+    # Remove the long term trend from the anomalies
+    detrendedAnomal = signal.detrend(anomaly.values)
```

Comment:
I did and get
```
Traceback (most recent call last):
  File "./run_analysis.py", line 243, in <module>
    analysis(config)
  File "./run_analysis.py", line 145, in analysis
    nino34_index(config)
  File "/global/u2/l/lvroekel/MPAS-Analysis/mpas_analysis/ocean/nino34_index.py", line 93, in nino34_index
    nino34 = compute_nino34_index(regionSST, calendar)
  File "/global/u2/l/lvroekel/MPAS-Analysis/mpas_analysis/ocean/nino34_index.py", line 157, in compute_nino34_index
    anomaly.apply(signal.detrend)
  File "/global/project/projectdirs/acme/software/anaconda_envs/edison_test/lib/python2.7/site-packages/xarray/core/common.py", line 168, in __getattr__
    (type(self).__name__, name))
AttributeError: 'DataArray' object has no attribute 'apply'
```
This is using @xylar's new conda environment with xarray 0.92.  Given this only removes one line of code, is it necessary to pursue this further?